frontend/src/screens/learner/details.tsx

```diff
@@ -95,6 +95,37 @@ const MultiSession: FC<StudyDetailsProps> = ({ study }) => {
     )
 }
 
+const StudyTime: FC<StudyDetailsProps> = ({ study }) => {
+    // TODO when https://github.com/openstax/research/issues/204 is complete
+    //  render this dynamically using study.stages
+    if (studyIsMultipart(study)) {
+        return (
+            <Box className='mb-1' direction='column'>
+                <Box gap>
+                    <Icon icon="clock" color={colors.purple} />
+                    <Box>
+                        <span>*Total: {study.durationMinutes}min</span>
+                        {study.participationPoints && <span>&nbsp;{study.participationPoints}pts</span>}
+                    </Box>
+                </Box>
+                <Box css={{ color: colors.grayText }} direction='column'>
+                    <small>Session 1: 20min 35pts</small>
+                    <small>Session 2: 20min 35pts</small>
```

Comment:
how about instead of hardcoding `20 min 35pts` we divide `study.durationMinutes / study.parts.length`?